gapic/src/main/com/google/gapid/perfetto/views/SlicesSelectionView.java

```diff
@@ -50,7 +50,7 @@
  * Displays information about a list of selected slices.
  */
 public class SlicesSelectionView extends Composite {
-  private static final int PROPERTIES_PER_PANEL = 8;
+  private static final int PROPERTIES_PER_PANEL = 9;
```

Comment:
I think it's ok to keep it as 8, this constant is to set how many rows of properties displayed for each column, since we are not changing the info box's default height, maybe we should also keep this constant unchanged?

Reply:
I made it 9 to accommodate the "General Info" label and avoid touching your custom layout logic for the next column. Currently there's a finite amount of handles we handle so we can safely avoid more complex logic for now.  Adding more logic to make it overflow nicely into the next column seems non-trivial, but can be done if you think it's important.